src/pushpa/CountSplUpperLowerCharFreq.java

```diff
@@ -0,0 +1,36 @@
+//Assignment - 23 : 14th Sep'2023
+//Write a program to find count of Uppercase, lowercase, digits and special characters.
+//
+//String str = "TecHNo_cR6ediTs";
+//output : Uppercase -> 5
+//         Lowercase -> 8
+//		   Digit -> 1
+//		   Special char count -> 1
+
+package pushpa;
+
+public class CountSplUpperLowerCharFreq {
+
+	public void countSplUpperLowerCharFreqency(String str) {
+		int upperCount = 0, lowerCount = 0, digitCount = 0, splCount = 0;
+		for (int i = 0; i < str.length(); i++) {
+			if (Character.isUpperCase(str.charAt(i)))
```

Comment:
It would be better if you create char ch = str.charAt(index) outside of condition, so you no need to write it again and again, you have to just write ch